new-site/docs-site/content/0.19.0/guide/typesense-dynamodb.md

```diff
@@ -0,0 +1,297 @@
+# DynamoDB Integration with Typesense
+
+Hey there! This post give you a guide to integrate Typesense custer with AWS DynamoDB by setting up a trigger with AWS Lambda.
+
+![Typesense DynamoDB Integration Chart](~@guide-images/typesense-dynamodb/dynamodb-typesense-integration-1.png)
+
+## Create AWS DynamboDB with streams enabled
+
+Create a DynamoDB with your choice of name and partition key ("id" is recommended). Now, after creating a DynamoDB database you should enable streams in the "Overview" section.
```

Comment:
> Create a DynamoDB
> after creating a DynamoDB database

DynamoDB is the name of the service. So you'd create a DynamoDB _table_

> you should enable streams

Could be phrased as "You want to enable streams"